### PR TITLE
Refactor Eventing reconciler to allow multi-step actions

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Refactor Eventing reconciler to allow multi-step actions.

### CODE REVIEW

File: pkg/reconciler/instances/eventing/preaction/preaction.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package preaction
```
Comment (clebs) [inline pkg/reconciler/instances/eventing/preaction/preaction.go:1]: The code in this file and for `postaction` look almost identical, is this duplication meaningful?
Might be better to have one step looping logic and concentrate the differences on the specific steps + parametric initialisation.

Comment (marcobebway) [reply pkg/reconciler/instances/eventing/preaction/preaction.go:1]: You are right. Changed.

**Review by clebs — APPROVED** [review]

**Review by clebs — APPROVED** [review]

**Pull request merged**